lib/diagnostic.rb

```diff
@@ -29,6 +29,10 @@
 
 character = 'Jar Jar Binks'
 
+def ANSWER = 'Jar Jar Binks'
+
+  
+
 Diagnostic.interpolation = 'ANSWER is really a Sith Lord.'
```

Comment:
you have to call it. `Diagnostic.interpolation = "#{ANSWER} is really a Sith Lord."`